### PR TITLE
Added solution for ys-SortPointsByArgument

### PR DESCRIPTION
**Pull request by RedBlueBird**
Thanks to Dustin Miao's solution!

_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/ys-SortPointsByArgument.mdx
Comment (SansPapyrus683) [inline solutions/ys-SortPointsByArgument.mdx:10]: "based on their angle"
elaborate?

File: solutions/ys-SortPointsByArgument.mdx
Comment (SansPapyrus683) [inline solutions/ys-SortPointsByArgument.mdx:32]: just use a lambda

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/ys-SortPointsByArgument.mdx
Comment (RedBlueBird) [reply solutions/ys-SortPointsByArgument.mdx:10]: "we can find the angle of the point $P_i = (x_i,y_i)$ by calling $\tan^{-1}{\frac{y_i}{x_i}}$."
The angle of a point is referring to this sentence.

Comment (SansPapyrus683) [reply solutions/ys-SortPointsByArgument.mdx:10]: is that measurement actually called the angle of a point?

Comment (RedBlueBird) [reply solutions/ys-SortPointsByArgument.mdx:10]: Wait my bad, meant to say "angle of a point from the origin", now this is an actual thing

we can find the angle of the point $P_i = (x_i,y_i)$ from the origin by calling $\tan^{-1}{\frac{y_i}{x_i}}$. To sort the points in the counterclockwise order, we just sort the points based on their angles from the origin from smallest to largest. 

Is this better?

Comment (SansPapyrus683) [reply solutions/ys-SortPointsByArgument.mdx:10]: i'll wait for a second opinion actually

**Comment by SansPapyrus683** [comment]
jeff i'd love to merge but let's wait for another person's approval first

**Comment by RedBlueBird** [comment]
> jeff i'd love to merge but let's wait for another person's approval first

Ah make sense, Is it possible to get an approval at least so the status won't stay as "changes requested"

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/ys-SortPointsByArgument.mdx
Comment (ryanchou-dev) [inline solutions/ys-SortPointsByArgument.mdx:30]: kind of unnecessary?

File: solutions/ys-SortPointsByArgument.mdx
Comment (ryanchou-dev) [inline solutions/ys-SortPointsByArgument.mdx:44]: ^^

File: solutions/ys-SortPointsByArgument.mdx
Comment (ryanchou-dev) [inline solutions/ys-SortPointsByArgument.mdx:38]:
```suggestion
	// sort counterclockwise
```
optional

**Review by ryanchou-dev — APPROVED** [review]

File: solutions/ys-SortPointsByArgument.mdx
Comment (dongliu0426) [inline solutions/ys-SortPointsByArgument.mdx:45]: what's the point of `setprecision(0)`?

**Comment by dongliu0426** [comment]
https://judge.yosupo.jp/submission/76945 WA

File: solutions/ys-SortPointsByArgument.mdx
Comment (RedBlueBird) [reply solutions/ys-SortPointsByArgument.mdx:45]: Oh oops, forgot to remove that, setprecision(0) isn't necessary.

**Comment by RedBlueBird** [comment]
> https://judge.yosupo.jp/submission/76945 WA

Woah really. 
Ok it works now, https://judge.yosupo.jp/submission/76952 , try submitting it again

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**
